src/utils/SConscript

```diff
@@ -1,5 +1,6 @@
 """Build DAOS utilities"""
 import daos_build
+import os
```

Comment:
Is this intentional?  Why are we adding an import and not making use of it?

Reply:
done